ui-v2/src/components/concurrency/task-run-concurrenct-view/header.tsx

```diff
@@ -0,0 +1,22 @@
+import { Button } from "@/components/ui/button";
+import { Icon } from "@/components/ui/icons";
+import { Typography } from "@/components/ui/typography";
+
+type Props = {
+	onAdd: () => void;
+};
+
+export const TaskRunConcurrencyLimitsHeader = ({ onAdd }: Props) => {
+	return (
+		<div className="flex gap-2 items-center">
+			<Typography variant="h4">Task Run Concurrency Limits</Typography>
+			<Button
+				onClick={onAdd}
+				size="icon"
+				aria-label="add task run concurrency limit"
+			>
+				<Icon id="Plus" />
+			</Button>
```

Comment:
I think we should make this an outline button and smaller to match the current style.
```suggestion
			<Button
				onClick={onAdd}
				size="icon"
				className="h-7 w-7"
				aria-label="add task run concurrency limit"
				variant="outline"
			>
				<Icon id="Plus" className="h-4 w-4" />
			</Button>
```